packages/pn-commons/package.json

```diff
@@ -6,11 +6,11 @@
   "dependencies": {
     "@date-io/date-fns": "^2.15.0",
     "@mui/icons-material": "^5.14.3",
-    "@mui/lab": "5.0.0-alpha.140",
-    "@mui/material": "5.14.5",
+    "@mui/lab": "^5.0.0-alpha.140",
```

Comment:
@AndreaCimini90 why the use of ^ for these two packages?

Reply:
To use the same that are used in @pagopa/mui-italia